guide/interactions/select-menus.md

```diff
@@ -0,0 +1,312 @@
+# Select menus
+
+With the components API, you can create interactive message components. In this page, we'll be focusing on how to send, receive, and respond to select menus using discord.js!
+
+::: danger
```

Comment:
danger components should only be used if there is an actual danger of messing up. if referring to earlier sections that this builds on top of, prefer tip elements